packages/flutter/test/widgets/list_wheel_scroll_view_test.dart

```diff
@@ -1195,7 +1195,7 @@ void main() {
       await tester.fling(
         find.byType(ListWheelScrollView),
         const Offset(0.0, -50.0),
-        800.0,
+        100.0,
```

Comment:
@grouma Can you recall why this fling needed to be reduced so much? This is probably the same regression that has been reported for flings going way too far.

Reply:
Unfortunately I do not. I recall that a number of tests had to have their parameters updated to keep the "goldens" valid. The main intent behind this change was to fix the jitter users would experience as the scroll slowed down. I added a test for that invalid behavior, see `ClampingScrollSimulation velocity eventually reaches zero`.